### PR TITLE
take usage tile off dashboard and add to account page

### PR DESCRIPTION
**Pull request by whimsicallyson**
PR Checklist:

- [x] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
moves usage tile off of main dashboard so you're not constantly seeing the numbers tick away

### CODE REVIEW

File: src/pages/Dashboard/Dashboard.vue [hunk 1]
```diff
@@ -124,24 +118,6 @@ export default {
...
-      if (!this.isCloud) return null
-      if (!this.license) return 'loading'
-      const isSelfServe = this.license.terms.is_self_serve
-      const isUsageBased = this.license.terms.is_usage_based
-
-      // Legacy license, not self-serve (so no upgrade)
-      if (!isSelfServe && !isUsageBased) return null
-
-      // Legacy license, self-serve (so can upgrade)
-      if (isSelfServe && !isUsageBased) return 'UpgradeUsageTile'
```
Comment (znicholasbrown) [inline src/pages/Dashboard/Dashboard.vue:137]: I think it'd still be good to show the upgrade tile here (and on the account page)

**Review by znicholasbrown — CHANGES_REQUESTED** [review]

File: src/pages/Dashboard/Dashboard.vue (hunk 1 above)
Comment (whimsicallyson) [reply src/pages/Dashboard/Dashboard.vue:137]: should be good now :)

**Review by znicholasbrown — CHANGES_REQUESTED** [review]
I think the usage tile needs to be moved above the payment tile and the title should match those on the rest of the page; the details button can also be removed, since it links to the page this is already on.

File: src/pages/Dashboard/UsageTiles/CycleUsage-Tile.vue [hunk 2]
```diff
@@ -93,10 +93,12 @@ export default {
     style="height: 100%;"
     tile
   >
+    <v-card-title class="text-h4 font-weight-light"
+      >Usage this cycle</v-card-title
+    >
     <v-card-text
       class="pa-0 px-3 pt-3 d-flex align-start justify-center flex-column mb-auto"
```
Comment (znicholasbrown) [inline src/pages/Dashboard/UsageTiles/CycleUsage-Tile.vue:100]:
```suggestion
      class="pa-3 pt-0 d-flex align-start justify-center flex-column mb-auto"
```

**Review by znicholasbrown — CHANGES_REQUESTED** [review]
Looks good! Just a padding fix suggestion

File: src/pages/Dashboard/UsageTiles/CycleUsage-Tile.vue (hunk 2 above)
Comment (whimsicallyson) [reply src/pages/Dashboard/UsageTiles/CycleUsage-Tile.vue:100]: fine by me!

**Review by zhen0 — APPROVED** [review]
This looks good to me but I'll let you have final review/merge @znicholasbrown

**Review by znicholasbrown — APPROVED** [review]
Yup LGTM 👍

**Pull request merged**